draft-gpew-priv-ppm.md

```diff
@@ -1442,35 +1442,62 @@ defined by the PPM parameters. (See {{batch-parameter-validation}}.)
 
 # Security Considerations {#sec-considerations}
 
-Prio assumes a powerful adversary with the ability to compromise an unbounded
-number of clients. In doing so, the adversary can provide malicious (yet
-truthful) inputs to the aggregation function. Prio also assumes that all but one
-server operates honestly, where a dishonest server does not execute the protocol
-faithfully as specified. The system also assumes that servers communicate over
-secure and mutually authenticated channels. In practice, this can be done by TLS
-or some other form of application-layer authentication.
-
-In the presence of this adversary, Prio provides two important properties for
-computing an aggregation function F:
-
-1. Privacy. The aggregators and collector learn only the output of F computed
-   over all client inputs, and nothing else.
-1. Robustness. As long as the aggregators execute the input-validation protocol
-   correctly, a malicious client can skew the output of F only by reporting
-   false (untruthful) input. The output cannot be influenced in any other way.
-
-There are several additional constraints that a Prio deployment must satisfy in
-order to achieve these goals:
-
-1. Minimum batch size. The aggregation batch size has an obvious impact on
-   privacy. (A batch size of one hides nothing of the input.)
-2. Aggregation function choice. Some aggregation functions leak slightly more
-   than the function output itself.
-
-[TODO: discuss these in more detail.]
+PPM assumes an active attacker that controls the network and has the ability to
```

Comment:
```suggestion
DAP assumes an active attacker that controls the network and has the ability to
```
Since #220

Reply:
This change needs to be applied elsewhere, better to do it it one change I think.